.github/workflows/build.yml

```diff
@@ -89,27 +89,34 @@ jobs:
           CHISEL_VERSION: ${{ steps.build.outputs.CHISEL_VERSION }}
         run: |
           ARCHIVE_FILE=chisel_${CHISEL_VERSION}_${GOOS}_${GOARCH}.tar.gz
+          ARCHIVE_FILE_CKSUM="${ARCHIVE_FILE}.sha384"
           echo "Creating archive $ARCHIVE_FILE"
 
           mkdir -p dist/build
           cp chisel LICENSE README.md dist/build
           find dist/build -printf "%P\n" | tar -czf dist/$ARCHIVE_FILE --no-recursion -C dist/build -T -
+          cd dist/ && sha384sum "${ARCHIVE_FILE}" > "${ARCHIVE_FILE_CKSUM}" && cd -
 
           # Share variables with subsequent steps
           echo "ARCHIVE_FILE=${ARCHIVE_FILE}" >>$GITHUB_OUTPUT
+          echo "ARCHIVE_FILE_CKSUM=${ARCHIVE_FILE_CKSUM}" >>$GITHUB_OUTPUT
```

Comment:
we probably should improve the code maintainability here, cause `dist/` is hardcoded throughout 3 different steps.

It would be better to simply include `dist/` as part of the variable names such that the following artifacts and release assets don't need to assume that prefix.

Reply:
Good point. Fixed in https://github.com/canonical/chisel/pull/155/commits/b2fa5d521dcbfe2d602e26160d3f418f4087773a. Instead of placing the tarballs in `dist/`, placing the tarballs in workdir.